common/configurable/src/main/java/io/helidon/common/configurable/ThreadPoolSupplier.java

```diff
@@ -171,7 +198,8 @@ private Builder() {
         @Override
         public ThreadPoolSupplier build() {
             if (name == null) {
-                name = DEFAULT_POOL_NAME_PREFIX + DEFAULT_NAME_COUNTER.incrementAndGet();
+                LOGGER.warning("Neither a thread name prefix nor a pool name specified");
```

Comment:
Is this message correct? If I specify a thread name prefix but no pool name won't this message still be logged?

Reply:
Good catch, looks like I removed an if statement by accident.